.github/tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
     "compilerOptions": {
         "module": "commonjs",
-        "target": "es2020",
+        "target": "ES2021",
```

Comment:
Let's use `ES2023`, because why not prevent someone from encountering confusion when using another newer JS api? For example, `ES2023` contains `array.findLast`

Reply:
Also, can we make the same change in `scripts/tsconfig.json`?